src/SonarQube.TeamBuild.PostProcessor/Resources.resx

```diff
@@ -202,4 +202,7 @@ Config file: {5}</value>
   <data name="WARN_CannotProcessCoverage" xml:space="preserve">
     <value>Cannot process the code coverage information. Look at the previous warnings for details.</value>
   </data>
+  <data name="WARN_XamlBuildDeprecated" xml:space="preserve">
+    <value>WARNING: Support for XAML builds is deprecated and will be removed in a future release of the Scanner for MSBuild</value>
```

Comment:
Do we want to state it'll be version 5?

Reply:
@michalb-sonar It seems that it's not the option people prefer. Look at the email thread.